deployments/k8s/helm/signalfx-agent/values.yaml

```diff
@@ -263,6 +263,11 @@ hostFSPath: /hostfs
 # monitors will be configured.
 configureStandardMonitors: true
 
+# If true, this will set the `perCPU` config option to `true` on `load`
+# standard monitor so the metric will be the average load per CPU.
+# Has no effect if `configureStandardMonitors: false`
+configureLoadPerCPU: false
```

Comment:
Let's just call this `loadPerCPU` without the `configure` prefix.

Reply:
done